src/components/Button/Button.scss

```diff
@@ -247,27 +247,58 @@ $partial-button-filled-pressed-box-shadow: inset 0 0 0 0 transparent,
 // stylelint-disable selector-max-specificity
 
 .plain {
-  @include recolor-icon(color('blue'));
+  @include recolor-icon(var(--p-interactive-action, color('blue')));
   margin: (-1 * $vertical-padding) rem(-8px);
   padding-left: spacing(tight);
   padding-right: spacing(tight);
   background: transparent;
   border: 0;
   box-shadow: none;
-  color: color('blue');
+  color: var(--p-interactive-action, color('blue'));
+
+  .Content {
+    font-weight: 400;
+  }
 
   &.pressed,
   &:hover,
   &:focus,
   &:active {
-    @include recolor-icon(color('blue', 'dark'));
     background: transparent;
     border: 0;
     box-shadow: none;
+  }
+
+  // TODO: implement pressed for masterbrand
```

Comment:
Does this mean the interaction states? Also, maybe refer to it as global theming.

Reply:
```suggestion
  // TODO: implement pressed for global theming
```

Reply:
This means the 'depressed' state for `Button` that was recently added. I wanted to punt on implementing that until we have designs nailed down, and the right color variants to cover it.